service_container.rst

```diff
@@ -133,68 +133,61 @@ the service container *how* to instantiate it:
 
         # app/config/services.yml
         services:
-            app.message_generator:
-                class:     AppBundle\Service\MessageGenerator
-                arguments: []
+            # configures defaults for all services in this file
+            _defaults:
+                autowire: true
+                autoconfigure: true
```

Comment:
this isn't merged yet - symfony/symfony#22234 - but has all the 👍 so I expect it will